scripts/create_tables.sql

```diff
@@ -1 +1,28 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors(
+  id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER VARYING,
+  dob DATE
+  specialty CHARACTER VARYING
+);
+
+CREATE TABLE patients(
+  id SERIAL PRIMARY KEy,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER VARYING,
+  dob DATE
+  insurance CHARACTER VARYING
+);
+
+CREATE TABLE appointments(
+  id SERIAL PRIMARY KEY,
+  date_of TEXT,
+  time TEXT,
```

Comment:
I'm not sure if `time` might be a restricted word in SQL, so this might not work, but I won't take points off for it.